cmd/tfimporter/README.md

```diff
@@ -0,0 +1,40 @@
+# Terraform Importer
+
+Status: Early Access Program
+
+Terraform Importer is a standalone utility to import existing infrastructure
+into a Terraform state, based on existing Terraform configs. Its target use-case
+is to facilitate migration between other infrastructure-as-code tools (such as
+[Deployment Manager](https://cloud.google.com/deployment-manager)), and configs
+generated by the Terrarom Engine.
+
+## Requirements
+
+-   [Terraform 0.12+](https://www.terraform.io/downloads.html)
+-   [Go 1.14+](https://golang.org/dl/)
+
+## Usage
+
+```
+$ go install ./cmd/tfimporter
+$ tfimporter -- --input_dir ~/path/to/terraform/configs/dir/
```

Comment:
You don't need the extra -- anymore

Also, it may be cleaner to call this just `tfimport`

In the future I would expect to do:
$ cd some-dir
$ terraform init
$ tfimport
$ terraform apply

Reply:
Removed -- and renamed to tfimport.

And the importer actually runs init, so it'd be more like 1. Generate 2. Import 3. Plan 4. Apply